app/assets/stylesheets/common/base/login-signup-page.scss

```diff
@@ -0,0 +1,295 @@
+// Shared styles
+
+.login-fullpage,
+.signup-fullpage,
+.invites-show {
```

Comment:
I'm not sure whether it is related to this CSS here, but the invites page is broken on this branch. Screenshot: 

<img width="1330" alt="image" src="https://github.com/user-attachments/assets/b6eea908-8a43-45ee-90ec-01e7256961a5">

Reply:
Sorry, this is still broken for me locally @jancernik.

Reply:
The invite and everything seem the same, but it works for me. I changed the styles to not use flex at all for invites, maybe now it works...